Dockerfile

```diff
@@ -76,7 +76,19 @@ RUN if [ "$TARGETPLATFORM" = "linux/amd64" ]; then \
   && dpkg -i /tmp/simba/*.deb \
   && printf "[Simba]\nDriver = /opt/simba/spark/lib/64/libsparkodbc_sb64.so" >> /etc/odbcinst.ini \
   && rm /tmp/simba_odbc.zip \
-  && rm -rf /tmp/simba; fi
+  && rm -rf /tmp/simba; fi \
+  && curl "https://github.com/FirebirdSQL/firebird/releases/download/v5.0.0/Firebird-5.0.0.1306-0-linux-x64.tar.gz" --location --output /tmp/firebird.tar.gz \
+  && mkdir /tmp/firebird \
+  && tar -xvf /tmp/firebird.tar.gz --directory /tmp/firebird \
+  && mkdir /tmp/firebird/buildroot \
+  && tar -xvf /tmp/firebird/Firebird-*/buildroot.tar.gz --directory /tmp/firebird/buildroot \
+  && cp /tmp/firebird/buildroot/opt/firebird/lib/libfbclient.so /usr/lib/libfbclient.so.2 \
+  && ln -s /usr/lib/libfbclient.so.2 /usr/lib/libfbclient.so \
+  && cp /tmp/firebird/buildroot/opt/firebird/lib/libtomcrypt.so /usr/lib/libtomcrypt.so.1 \
+  && ln -s /usr/lib/libtomcrypt.so.1 /usr/lib/libtomcrypt.so \
+  && rm /tmp/firebird.tar.gz \
+  && rm -rf /tmp/firebird \
+  && ln -s /usr/lib/x86_64-linux-gnu/libtommath.so.1 /usr/lib/x86_64-linux-gnu/libtommath.so.0
```

Comment:
Does this really need to be installed for a firebird connector to work? I'd imagine we can just ping an existing firebird db to query it - I don't think we'd have to actually install it in the redash image.

Reply:
Unfortunately, we need some client libraries to be able to connect to a Firebird database.
See the [docs](https://firebirdsql.org/file/documentation/html/en/firebirddocs/qsg5/firebird-5-quickstartguide.html#qsg5-installing-client) for more info.